include/ccl_utils.h

```diff
@@ -15,6 +15,10 @@ extern "C" {
  */
 double * ccl_linear_spacing(double xmin, double xmax, int N);
 
+/**
+ * Compute bin edges of N-1 logarithmically and then linearly spaced bins on the interval [xmin,xmax]
```

Comment:
The header of the function in ccl_utils.h needs to follow the format of the other headers, so that the docs are processed automatically.

Reply:
Ok, I have changed the header of the function in include/ccl_utils.h.